packages/components/components/elvis-progress-linear/src/react/elvia-progress-linear.tsx

```diff
@@ -17,6 +18,17 @@ export const ProgressLinear: FC<ProgressLinearProps> = ({
   inlineStyle,
   ...rest
 }) => {
+  const lang = useLanguage();
+  const [computedAriaLabel, setComputedAriaLabel] = useState('');
+  const [computedAriaValueText, setComputedAriaValueText] = useState('');
+
+  useEffect(() => {
+    const newAriaLabel = lang === 'no' ? 'Progresjon' : 'Progress';
+    const newAriaValueLabel = lang === 'no' ? 'Progresjonen er nå på ' : 'The progress is now at ';
+    setComputedAriaLabel(newAriaLabel);
+    setComputedAriaValueText(newAriaValueLabel + value + '%.');
```

Comment:
I generally find it easier to read/use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) over simple string concatenation 🧹 
```suggestion
    setComputedAriaValueText(`${newAriaValueLabel} ${value}%.`);
```

Reply:
Can we calculate this aria label during rendering as suggested [here](https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state)?

> When something can be calculated from the existing props or state, [don’t put it in state.](https://react.dev/learn/choosing-the-state-structure#avoid-redundant-state) Instead, calculate it during rendering.

Reply:
It would be totally fine to remove the `useState` and `useEffect`s related to these states and just do them "inline" (as normal variables instead of `useState`) to be calculated during rendering, yes! That's probably a cleaner way of doing it too!

Reply:
Agreed, I think that will be better.

Reply:
Updated the code to use inline language selection based on `lang` attribute instead of the useEffect as it served little to no purpose.